helpers/util.py

```diff
@@ -58,5 +58,11 @@ def get_used_memory():
     return used_memory
 
 
+def numa_nodes_count():
+    cmd = f"lscpu | grep -oP 'NUMA node\(s\):\s*\K\d+'"
```

Comment:
I think you should move this function to `helpers.remote.INode` class as protected method. For example:
```
class INode(object, metaclass=abc.ABCMeta):
    def __init__(self, ntype: str, hostname: str, workdir: str, *args, **kwargs):
        ...
        self.numa_nodes_n: int = self._get_numa_nodes_count()

    def _get_numa_nodes_count() -> int:
        ...
```
Then we will always have access to the value (for each node) via `remote.client.numa_nodes_n`, `remote.tempesta.numa_nodes_n`,